test/test_restful.py

```diff
@@ -915,6 +946,49 @@ def test_060_cv_version_get(self):
         self.assertEqual(results['current_version'], api_version.current_version())
         self.assertEqual(results['supported_versions'], api_version.all_versions())
 
+    # Rust agent testset
+    @unittest.skipIf(RUST_TEST, "Testing against rust-keylime is disabled!")
+    def test_070_rust_agent_setup(self):
+        """Set up the Rust agent"""
+
+        # Kill the Python agent and launch the Rust agent!
+        kill_cloudagent()
+        launch_rust_cloudagent()
+        time.sleep(10)
```

Comment:
this second sleep can be skipped, if you use the "wait" loop. In any case there were a sleep inside the launch command.

Reply:
See above